### PR TITLE
Review 2

### PR DESCRIPTION
**Pull request by KanVull**
Added show product page

### CODE REVIEW

**Comment by KanVull** [comment]
There will be commits with letter_opener gem

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ProductsController < ApplicationController
+  def show
+    @product = Product.find_by!(id: params[:id])
```
Comment (igor-alexandrov) [inline app/controllers/products_controller.rb]: Lets replace this with `find`.

**Review by igor-alexandrov — CHANGES_REQUESTED** [review]

**Comment by KanVull** [comment]
This order link is not secure. It would probably be nice to create a unique id (like: kasdjf3434lkfjkld) for order and give the access to order by link our_site.com/orders/kasdjf3434lkfjkld

**Comment by KanVull** [comment]
Added UUID to order, returned id, fixed mailer, fixed variable scope set_cart method to private.

**Pull request merged**